src/js/base/module/ImageDialog.js

```diff
@@ -55,7 +55,6 @@ export default class ImageDialog {
   bindEnterKey($input, $btn) {
     $input.on('keypress', (event) => {
       if (event.keyCode === key.code.ENTER) {
-        event.preventDefault();
```

Comment:
Do you have any reason for removing preventDefault?
This was pushed by https://github.com/summernote/summernote/pull/2593 for preventing form submitting by hitting enter on dialogs.

Reply:
I didn't edit that, I'm not sure why my PR removed it.